pkg/machinery/client/client.go

```diff
@@ -302,7 +307,7 @@ func NewClient(cfg *tls.Config, endpoints []string, port int, opts ...grpc.DialO
 	c = &Client{}
```

Comment:
can we please drop `NewClient` method altogether?

Reply:
While we're at it, it looks to me like NewClientContextAndCredentialsFromConfig and NewClientContextAndCredentialsFromParsedConfig are also depreceated and unused. Should we delete those aswell or do other repositories/users depend on it?

Reply:
a quick search of github says they aren't used anywhere

Reply:
done